cms/templates/course_outline.html

```diff
@@ -1,6 +1,6 @@
 <%page expression_filter="h"/>
 <%inherit file="base.html" />
-<%def name="online_help_token()"><% return "outline" %></%def>
+<%def name="online_help_token()"><% return "develop_course" %></%def>
```

Comment:
@catong This help token isn't currently being used at all, I just wanted to provide a sane default in case some other link gets added to this page that wants to use the existing `${get_online_help_info(online_help_token())['doc_url']}` pattern. We can completely remove this line if you don't think that'll be an issue.